config/locales/en.yml

```diff
@@ -336,7 +336,7 @@ en:
       rotate_certificate_read_steps: The process for rotating each certificate is slightly different. Read the steps on each page carefully, even if you’ve rotated certificates before.
       MSA: Before you upload your Matching Service Adapter (MSA) %{type} certificate,
       VSP: Before you upload your Verify Service Provider (VSP) %{type} certificate,
-      SP: Before you upload your service provider %{type} certificate,
+      SP: 'Before you upload your service provider %{type} certificate:'
```

Comment:
This is something that should actually be brought up with @bethwalton? The `you must:` part doesn't appear to be in the service provider encryption journey at all but does with the VSP and MSA. So should it be consistent and be here too?

Reply:
I think it's because of the numberes steps, as on the non dual running flow one is must and the other one is a different sentence where the must doesn't fit and so it's within the first sentence...